### DISCUSSION TITLE
Movement of the point cloud map

### DISCUSSION THREAD

**Discussion by jvyou** [NVlabs/curobo#326] [Software Q&A]
Hello
We modified the URDF file of the Franka robotic arm to make the robotic arm move up by 0.2 as a whole, but we found that the point cloud map of the robotic arm did not move, it still followed the point cloud map to plan the grasping path, resulting in a 0.2 gap between the actual robotic arm and the red square on the Z axis, how to modify the initial position of the point cloud map, so that the point cloud map also moved up by 0.2, and finally made the robotic arm accurately grasp the red square in the view
![屏幕截图 2024-07-06 160159](https://github.com/NVlabs/curobo/assets/121914445/57e807bb-8400-4035-aba0-f330fc9eb97d)

**Comment by balakumar-s** [comment]
The spheres that you see are only used for debugging and they do not read the robot's pose w.r.t. world. They are spawned at the world origin.
